django_loci/base/admin.py

```diff
@@ -280,7 +280,7 @@ def _get_location_instance(self):
         data = self.cleaned_data
         location = data.get('location') or self.location_model()
         location.type = data.get('type') or location.type
-        location.is_mobile = data.get('is_mobile') or location.is_mobile
```

Comment:
@Purhan I don't support the idea of removing the `or` part of this statement because it must have been put there for some reason. this solution fixes the problem but it can cause issues in other modules which depends on this one. i think a good fix should be:
```location.is_mobile = location.is_mobile if data.get('is_mobile') is None else data.get('is_mobile')```

this fixes our problem and also maintain the original logic. 

What do you think?

Reply:
Thank you, I was also skeptical about that change beforehand :smile:. I will quickly make these changes.